src/VCR/Storage/AbstractStorage.php

```diff
@@ -104,6 +116,8 @@ public function isNew() {
      */
     public function __destruct()
     {
-        fclose($this->handle);
+        if (is_resource($this->handle)) {
+            fclose($this->handle);
+        }
```

Comment:
not directly related but i got very ugly console output when things go wrong in the Storage constructor.